### ISSUE TITLE
Installation fails with "Cannot read property 'length' of undefined"

### ISSUE THREAD

**Issue by Bikeman868** [psychobolt/react-pie-menu#7]
The component would not install with React v17, so I downgraded to v16, but I still cannot get this component to install.

This is the dependencies section of my `package.json` file:

```json
  "dependencies": {
    "@material-ui/core": "^4.11.0",
    "@material-ui/icons": "^4.9.1",
    "@material-ui/lab": "^4.0.0-alpha.56",
    "@slack/web-api": "^5.14.0",
    "d3": "^6.2.0",
    "d3-force": "^2.1.1",
    "pinch-zoom-js": "^2.3.4",
    "react": "^16.14.0",
    "react-dom": "^16.14.0",
    "react-is": "^17.0.1",
    "react-router-dom": "^5.2.0",
    "regenerator-runtime": "^0.13.7",
    "styled-components": "^4.4.1"
  },
```

I get an error message "Cannot read property 'length' of undefined" when I run the command `npm i react-pie-menu`.
I am running node v15.2.1

**Comment by psychobolt** [comment]
Hi, a new version with React 17 support is in the works.

**Comment by Bikeman868** [comment]
Good to hear. Any idea why I can't get it to install with React v16?

**Comment by psychobolt** [comment]
Haven't looked into, but I suspect you could also downgrade react-is and see if that resolves.

**Comment by Bikeman868** [comment]
I gave that a shot, but unfortunately its the same result.

```json
  "dependencies": {
    "@material-ui/core": "^4.11.0",
    "@material-ui/icons": "^4.9.1",
    "@material-ui/lab": "^4.0.0-alpha.56",
    "@slack/web-api": "^5.14.0",
    "d3": "^6.2.0",
    "d3-force": "^2.1.1",
    "pinch-zoom-js": "^2.3.4",
    "react": "^16.14.0",
    "react-dom": "^16.14.0",
    "react-is": "^16.13.1",
    "react-router-dom": "^5.2.0",
    "regenerator-runtime": "^0.13.7",
    "styled-components": "^4.4.1"
  },
```

**Comment by Bikeman868** [comment]
I upgraded from Node v15.2.1 to Node v15.3.0 and I get a whole lot of warnings but the component did install.
My website appears to still work too, so this maybe a working setup for me.

**Comment by psychobolt** [comment]
Hi, I just released [v0.3.0](https://github.com/psychobolt/react-pie-menu/tree/react-pie-menu%400.3.0). Now supports React 17.
